frontend/docs/application_config.md

```diff
@@ -211,6 +211,64 @@ To set them up, we'll use the current configuration objects for the resources. F
 
 The above code will show a notice with a red exclamation icon whenever a final user visits the table's Table Detail page or the Dashboard Detail page.
 
+```
+  resourceConfig: {
+    [ResourceType.table]: {
+      ... //Table Resource Configuration
+      notices: {
+          "<CLUSTER>.<DATABASE>.<SCHEMA>.*": {
+            severity: NoticeSeverity.ALERT,
+            messageHtml: `This table is deprecated`,
+          },
+      },
+    },
+    [ResourceType.dashboard]: {
+      ... //Dashboard Resource Configuration
+      notices: {
+          "<PRODUCT>.<CLUSTER>.<GROUPNAME>.*": {
+            severity: NoticeSeverity.WARNING,
+            messageHtml: `This dashboard is deprecated`,
+          },
+      },
+    },
+
+  },
+```
+
+The above code will show a notice with a red exclamation icon whenever a final user visits any table within the specified cluster, database, and schema or any dashboard within the specified product, cluster, and groupname.
```

Comment:
What we were doing of introducing the code after showing it might work with one example, but now it seems we would better introduce the code **before** we show it. Try something like:

If you want to target several tables at once, you can use wildcards as showed below:
//code
This will show a notice with a red...